theano/sandbox/cuda/cuda_ndarray.cuh

```diff
@@ -48,16 +54,19 @@
 #include <cublas_v2.h>
 
 #ifdef _WIN32
+/*
 # ifdef _CUDA_NDARRAY_C
 #  define DllExport   __declspec( dllexport )
 # else
 #  define DllExport   __declspec( dllimport )
 # endif
+*/
```

Comment:
I would delete this comment to make the code cleaner. and the line 66. Otherwise, it seem good for me, but @abergeron know this more then me.

Did you tried it with a clean Theano cache on windows?